module/VuFind/src/VuFind/View/Helper/Root/Config.php

```diff
@@ -88,4 +88,20 @@ public function ajaxCoversEnabled()
     {
         return $this->get('config')->Content->ajaxcovers ?? false;
     }
+
+    /**
+     * Get advanced search limits that override the natural sorting to
+     * display at the top.
+     *
+     * @param string $limit advanced search limit
+     *
+     * @return array
+     */
+    public function topLimits($limit)
+    {
+        $limits = $this->get('facets')->Advanced_Settings->limit;
+        $limitConf = $limits && $limits->get($limit) ?
+            preg_replace('/\s+/', '', $limits->get($limit)) : '';
+        return explode(',', $limitConf);
```

Comment:
It might be wise to have a configurable delimiter, or to take a different approach to multi-valued configuration, since it's not uncommon for facet values to include commas, and that would prevent certain facet fields (like author) from working well with this new feature.

Reply:
Great idea! I changed the default delimiter to `::` and made it configurable.